components/Avatar/Avatar.unit.test.jsx

```diff
@@ -0,0 +1,33 @@
+
+describe('<Avatar />', () => {
+  describe('Should match snapshot', () => {
+    it('With picture', () => {
+      expect(
+        renderer.create(<Avatar picture={encodedPicture} />).toJSON(),
+      ).toMatchSnapshot();
+    });
+
+    it('Without picture', () => {
+      expect(renderer.create(<Avatar />).toJSON()).toMatchSnapshot();
+    });
+
+    it('With notification', () => {
+      expect(
+        renderer.create(<Avatar hasNotification />).toJSON(),
+      ).toMatchSnapshot();
+    });
+
+    it('With text', () => {
+      expect(
+        renderer.create(<Avatar text="Profile" />).toJSON(),
+      ).toMatchSnapshot();
+    });
+  });
```

Comment:
What do you think to create a case using `Popover` component?

Reply:
the popover is not part of the component